content/academy/web_scraping_for_beginners/data_collection/devtools_continued.md

```diff
@@ -1,66 +1,65 @@
 ---
 title: DevTools continued
-description: Continue learning how to collect data from a website using browser DevTools, CSS selectors and JavaScript.
+description: Continue learning how to collect data from a website using browser DevTools, CSS selectors, and JavaScript via the DevTools console.
 menuWeight: 3
 paths:
     - web-scraping-for-beginners/data-collection/devtools-continued
 ---
 
 # [](#finish-devtools) Finish with DevTools
 
-In the first part of the DevTools lesson, we were able to collect information about a single website from the <a href="https://www.alexa.com/topsites" target="_blank">Alexa Top Sites index</a>. If you missed the previous part, we suggest going through it to understand the basic concepts, but you don't need any of the code from there, we will start from scratch.
+In the first part of the DevTools tutorial, we were able to collect information about a single website from our fake <a href="https://commerce-qd83plqbj-mstephen19.vercel.app/" target="_blank">"Morgan Webstore" e-commerce site</a>. If you missed the previous part, we suggest going through it to understand the basic concepts, but you don't need any of the code from there, we will start from scratch.
```

Comment:
Rename the Morgan Webstore to Fake Webstore. No need to say fake Morgan Webstore then. Maybe Fakestore? 😄

Reply:
Done!